speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Mediator.inl

```diff
@@ -90,45 +89,21 @@ tIndicator Mediator<DecisionMaker>::GetIndicatorSettings()
 template<typename DecisionMaker>
 void Mediator<DecisionMaker>::DriveTick(tCarElt* p_car, tSituation* p_situation)
 {
+    m_tickCount++;
```

Comment:
Placing this at the start of this function means we will start counting from 1, I'm not sure if we want this

Reply:
Sure I will change that